src/pages/export-page/ExportPage.tsx

```diff
@@ -32,7 +32,13 @@ export const ExportPage = () => {
         developmentPlanEvent === TimetableEventKey.TimetableUpdated
     );
     return [
-      ...timetableEvents,
+      ...timetableEvents.map((Event) => {
```

Comment:
lower case "event". My bad I think I had capitalised "Event" in my last comment 😅